infra/terraform/modules/log_pruning/prune_logs/requirements.txt

```diff
@@ -1,3 +1,3 @@
 certifi==2016.8.8
 elasticsearch-curator==4.0.6
-PyYAML==3.13
+PyYAML>=4.2b1
```

Comment:
I'd pin it to a known good version rather than `>=4.2b1` latest I can [see](https://pypi.org/project/PyYAML/#history) is `==4.2b4` but it feels weird to pin to a `b` version so maybe do something like `>=4.2b1,<5` so we don't inadvertently get moved to the next version

Reply:
Sure. I just put in what the vulnerability alert said.